docs/book/stacks-and-components/component-guide/writing-custom-components.md

```diff
@@ -0,0 +1,239 @@
+---
+description: How to extend ZenML with plug-in flavor components.
+---
+
+# Develop a Custom Component Flavor in ZenML
```

Comment:
Wonder if we should have some language which is a bit more open. Something like "use your own tools with ZenML" or something. Maybe out of place in this kind of documentation, but it might help someone to know at a quick glance what this page is about, or how to find what they're looking for when they don't know what component or flavor means in the context of ZenML.

Reply:
Can be in title or in subtitle or maybe just in how we link to the page?

Reply:
agree with Alex here, the title might not be immediatley obvious to newbees